frontend/src/components/EmbeddedGoogleSlides/index.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from "@emotion/styled";
+import { ReactElement } from "react";
+
+const ResponsiveContainer = styled.div`
+  position: relative;
+  overflow: hidden;
+  width: 100%;
+  padding-top: 56.25%;
+`;
+
+const ResponsiveIFrame = styled.iframe`
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  width: 100%;
+  height: 100%;
+`;
+
+const EmbeddedGoogleSlides = ({ src }: { src: string }): ReactElement => {
+  return (
+    <ResponsiveContainer>
+      <ResponsiveIFrame
+        sandbox="allow-scripts"
```

Comment:
Can we get security clearance on this please?

A quick glance looks unsafe, but professional opinion is needed 😆

https://stackoverflow.com/questions/35208161/is-it-safe-to-have-sandbox-allow-scripts-allow-popups-allow-same-origin-on-if

Thank you!